### PR TITLE
UsedVolumesSize and UsedStoreInBytes metrics 

### PR DESCRIPTION
**Pull request by klapkov**

### CODE REVIEW

**Comment by MarcPaquette** [comment]
This PR is related to https://github.com/cloudfoundry/garden-runc-release/issues/210

**Comment by MarcPaquette** [comment]
An internal tracking story has been created for this PR here: https://www.pivotaltracker.com/story/show/180559618

**Comment by MarcPaquette** [comment]
Hi @klapkov 
I have an issue logged in our backlog to have someone take a look at your PR.  At this time we are focusing on getting many of the Runtime resources bumped to 1.17.  That, coupled with the end of year holiday season, is slowing down our progress some.

Some quick feedback would be to squash the commits in the PR.

Thanks,
@MarcPaquette

File: groot/cleaner.go [hunk 1]
```diff
@@ -1,7 +1,9 @@
 package groot
 
 import (
+	"fmt"
 	"time"
+	"fmt"
```
Comment (MarcPaquette) [inline groot/cleaner.go:6]: Double `fmt` here. Could you remove one?

Comment (klapkov) [reply groot/cleaner.go]: Removing it right away.

**Review by MarcPaquette — COMMENTED** [review]
All these changes look good.

I'm going to confirm these changed with our PM.

**Comment by MarcPaquette** [comment]
Reviewed these changes with PM and Designer. Doing one final test, then merging in.

**Comment by MarcPaquette** [comment]
Hi @klapkov 

The metrics being emitted are being converted into a number that's non-human readable.   Would converting to MB or KB make more sense in this context given the sizes that may be present?

```
origin:"grootfs" eventType:ValueMetric timestamp:1644254494727066562 deployment:"cf" job:"diego-cell" index:"fd8b68ec-7582-4733-9a35-b8cbd1a890e7" ip:"10.0.1.11" tags:<key:"source_id" value:"grootfs" > valueMetric:<name:"UsedBackingStoreInBytes" value:2.700161024e+09 unit:"bytes" >
origin:"grootfs" eventType:ValueMetric timestamp:1644254494745729516 deployment:"cf" job:"diego-cell" index:"fd8b68ec-7582-4733-9a35-b8cbd1a890e7" ip:"10.0.1.11" tags:<key:"source_id" value:"grootfs" > valueMetric:<name:"UsedLayersSize" value:1.058217682e+09 unit:"bytes" >
origin:"grootfs" eventType:ValueMetric timestamp:1644254495438149477 deployment:"cf" job:"diego-cell" index:"fd8b68ec-7582-4733-9a35-b8cbd1a890e7" ip:"10.0.1.11" tags:<key:"source_id" value:"grootfs" > valueMetric:<name:"UsedLayersSize" value:1.058217682e+09 unit:"bytes" >
```

Thanks,
Marc

**Comment by klapkov** [comment]
Hi @MarcPaquette, 
Yes indeed currently they aren't non-human readable. When implementing the metrics, I noticed that all of the other ones are also emitted in bytes and just decided that the best choice is to follow the convention, but honestly from my point of view it doesn't matter if they are emitted in MB or KB. So I leave this decision to you guys. Should I change them ?

**Comment by MarcPaquette** [comment]
Hi @klapkov 

I don't have strong feelings either way. We can always change it down the line fairly easily.

 Thanks for the contribution and your patience!

**Pull request merged**